docs/cinder-csi-plugin/features.md

```diff
@@ -92,3 +93,9 @@ To avail the multiattach feature of cinder, specify the ID/name of cinder volume
 This volume type must exist in cinder already (`openstack volume type list`)
 
 This should enable to attach a volume to multiple hosts/servers simultaneously.
+
+## Liveness probe
+
+The [liveness probe](https://github.com/kubernetes-csi/livenessprobe) is a sidecar container that exposes an HTTP /healthz endpoint, which serves as kubelet's livenessProbe hook to monitor health of a CSI driver.
```

Comment:
Thanks looks good. Could you link it from main doc as well? https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/cinder-csi-plugin/using-cinder-csi-plugin.md#supported-features